docs/classes/sap_cloud_sdk_core.booleanfilterfunction.html

```diff
@@ -70,7 +70,7 @@ <h1>Class BooleanFilterFunction&lt;EntityT&gt;</h1>
 			<section class="tsd-panel tsd-comment">
 				<div class="tsd-comment tsd-typography">
 					<div class="lead">
-						<p>Representation of a filter function, that has returns a value of type boolean.</p>
+						<p>Representation of a filter function, that returns a value of type boolean.</p>
```

Comment:
I guess you regenerated the docs.
But from my point view, the docs should reflect the stable version instead of canary versions.
Therefore, only release pipeline should update the docs.

Reply:
Good catch, I will remove that.

Reply:
Actually, I just remembered, that I did not regenerate, I just fixed the grammar. Do you think it is fine to leave it? I could remove it otherwise.